demo/src/main/java/com/novoda/demo/MainActivity.java

```diff
@@ -75,17 +75,20 @@ public void onClick(View v) {
     };
 
     private final View.OnClickListener showAudioSelectionDialog = new View.OnClickListener() {
-
         @Override
         public void onClick(View v) {
-            dialogCreator.showAudioSelectionDialog();
+            if (player == null || !player.isPlaying() || player.getAudioTracks().size() == 0) {
+                Toast.makeText(MainActivity.this, "no audio tracks available!", Toast.LENGTH_LONG).show();
+            } else {
+                dialogCreator.showAudioSelectionDialog();
+            }
         }
     };
 
     private final View.OnClickListener showSubtitleSelectionDialog = new View.OnClickListener() {
         @Override
         public void onClick(View v) {
-            if (player.getSubtitleTracks().isEmpty()) {
+            if (player == null || !player.isPlaying() || player.getSubtitleTracks().isEmpty()) {
```

Comment:
Same question here (I know that this is a demo, I am just curious)

Reply:
Same as above 😄